### DISCUSSION TITLE
Custom rss/json api

### DISCUSSION THREAD

**Discussion by 0wx** [dailydotdev/daily#494] [Feature Requests]
I would love to get update with custom rss or json api to my feed in the extension

**Comment by idoshamun** [comment]
Hi @0wx,

Thanks for the suggestion.
Unfortunately, this is currently out of our scope.

**Reply by 0wx** [reply]
You can make this premium feature tho, I love your extension, and I'm willing to pay the premium for no ads and custom rss/json.

**Reply by idoshamun** [reply]
We actually had this as a premium feature including no ads as you mentioned. But it didn't work out.

**Reply by 0wx** [reply]
So you drop these premium feature? Too bad 😞 I'm willing to help creating this working out if you want.

**Reply by idoshamun** [reply]
Yap. Dropped them and never looked back. We try to focus on other aspects now.

**Reply by 0wx** [reply]
Alright then, thanks for everything you did btw, you did a great job for community 👍

**Reply by idoshamun** [reply]
Thank you!

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
